### PR TITLE
doc: document how to renew the docker-mirror-proxy CA certificate

### PR DESCRIPTION
**Pull request by brianmcarey**
**What this PR does / why we need it**:

Document how to renew the docker-mirror-proxy CA certificate

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @dhiller @xpivarc 

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/how-to-renew-docker-mirror-proxy-ca-cert.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Renewing the CA Certificate for the docker-mirror-proxy
+
+Prow jobs will fail to pull container images if the docker-mirror-proxy CA certificate is allowed to expire. 
+
+There is a [periodic job](https://github.com/kubevirt/project-infra/blob/dcd4fd00a2c1592e0db577bd824314cdd4221818/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L541) in place to check that the CA certificate will not expire within the next 90 days.
```
Comment (dhiller) [inline docs/how-to-renew-docker-mirror-proxy-ca-cert.md:5]: nit: add a link to the script referring to this doc page

**Review by dhiller — APPROVED** [review]
/approve

/hold for one nit inline

File: docs/how-to-renew-docker-mirror-proxy-ca-cert.md (hunk 1 above)
Comment (brianmcarey) [reply docs/how-to-renew-docker-mirror-proxy-ca-cert.md:5]: I added the document to the script.

**Review by dhiller — APPROVED** [review]
Thanks @brianmcarey !

/unhold

**Pull request merged**